### DISCUSSION TITLE
Checking list item with or without quotes not working antlr4

### DISCUSSION THREAD

**Discussion by techpranav** [antlr/antlr4#4174] [Q&A]
Hi,
Why below rules are not working as expected.
Its not matching "action" with quotes.

actionTypes: actionType_list
     | SINGLE_QUOTE actionType_list SINGLE_QUOTE
     | DOUBLE_QUOTE actionType_list DOUBLE_QUOTE
;

actionType_list :
    TRIGGER_TYPE_ACTION
    |	TRIGGER_TYPE_CHECK
    |   TRIGGER_TYPE_PRECOMMIT ;

**Comment by jimidle** [comment]
See my answer to your discussion question.
